extensions/hibernate-spatial/runtime/src/main/java/io/quarkus/hibernate/spatial/HibernateSpatialBuildTimeConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.hibernate.spatial;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.hibernate-spatial")
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public interface HibernateSpatialBuildTimeConfig {
```

Comment:
IIUC this is only needed during the build, so this class could probably live in the `deployment` module.

Though this might not be relevant anymore if you end up moving it all inside of the `hibernate-orm` extension :)